### DISCUSSION TITLE
Do qutebrowser has "Secure DNS"?

### DISCUSSION THREAD

**Discussion by 5KS** [qutebrowser/qutebrowser#6577] [Q&A] [answered]
Hello.
Do qutebrowser has Secure DNS?  Can it be disabled?

BG:
I'd set-up a dnscrypt-proxy2 resolver's cloaking, to map some IPs to *.domain.name.
I can not open these aliases in qutebrowser and Chromium, but can in Firefox and Epiphany.

I avoided the issue on mobile phone by turning OFF the Secure DNS feature, in Brave.
I did not found a setting for it in qutebrowser.

Maybe my issue is hidden elsewhere, though.

**Comment by The-Compiler** [comment, marked as answer]
Not entirely sure, and it'll most likely depend on your QtWebEngine version. Networking is entirely handled by the underlying Chromium.

Can you try launching with `--temp-basedir --qt-flag disable-features=DnsOverHttps` and see if that changes anything?

**Comment by 5KS** [comment]
Thanks.

Tried, failed.

Installed some more Chromium-based browsers:  with or without DoH it's same:  either 'NOT_RESOLVED' or 'NXDOMAIN'.
